audio/wav/decode.go

```diff
@@ -215,16 +215,14 @@ chunks:
 			}
 			channelCount := int(buf[2]) | int(buf[3])<<8
 			switch channelCount {
-			case 1:
-				mono = true
-			case 2:
-				mono = false
+			case 1, 2:
+				mono = channelCount == 1
 			default:
 				return nil, fmt.Errorf("wav: number of channels must be 1 or 2 but was %d", channelCount)
 			}
 			bitsPerSample = int(buf[14]) | int(buf[15])<<8
-			if bitsPerSample != 8 && bitsPerSample != 16 {
-				return nil, fmt.Errorf("wav: bits per sample must be 8 or 16 but was %d", bitsPerSample)
+			if !convert.IsValidResolution(bitsPerSample) {
+				return nil, fmt.Errorf("wav: invalid bits per sample must be [8,16,24,32] but was %d", bitsPerSample)
```

Comment:
Adding "invalid" here makes this error message sound unnatural. Better message: "wav: bits per sample must be 8, 16, 24, or 32, but was %d"

Reply:
okey dokey.